searx/engines/__init__.py

```diff
@@ -67,61 +53,57 @@ def load_engine(engine_data):
         engine_name = engine_name.lower()
         engine_data['name'] = engine_name
 
+    # load_module
     engine_module = engine_data['engine']
-
     try:
         engine = load_module(engine_module + '.py', engine_dir)
     except (SyntaxError, KeyboardInterrupt, SystemExit, SystemError, ImportError, RuntimeError):
         logger.exception('Fatal exception in engine "{}"'.format(engine_module))
         sys.exit(1)
-    except:
+    except BaseException:
         logger.exception('Cannot load engine "{}"'.format(engine_module))
         return None
 
+    # set engine attributes from engine_data
     for param_name, param_value in engine_data.items():
-        if param_name == 'engine':
-            pass
-        elif param_name == 'categories':
+        if param_name == 'categories':
             if param_value == 'none':
```

Comment:
Do we support category "none" .. I haven't found any usage of a string "none" in settings.yml nor in one of the `engines/*.py`